src/add-remove.js

```diff
@@ -0,0 +1,23 @@
+import { dataBase, updateValues, updateDatabase } from './functions.js';
+
+export function addTask(task) {
+  dataBase.push(task);
+  updateValues();
+}
+
+function normalizeIndexes() {
+  for (let x = 0; x < dataBase.length; x += 1) {
+    dataBase[x].index = x + 1;
+  }
```

Comment:
I believe it would be great to use a ```forEach``` loop to follow [KISS best practices](https://medium.com/infancyit/kiss-keep-it-simple-smart-javascript-habits-b019ce14b5ec).